public/components/integrations/components/integration.tsx

```diff
@@ -67,7 +67,7 @@ export function Integration(props: AvailableIntegrationProps) {
     } else {
       payload.index_patterns = [dataSourceName];
       return fetch(
-        `/api/console/proxy?path=_index_template/${componentName}_${version}&method=POST`,
+        `/api/console/proxy?path=_index_template/ss4o_${componentName}_${version}_${integration.name}&method=POST`,
```

Comment:
why are we creating a unique component for each integration ?
we should not duplicate similar schema components (http, cloud, communication )
the code should first check is the component exists and verify it has the same version - if does nothing to update 
Otherwise - add the new template with (including the version as part of the template name)

Reply:
The part that creates non-duplicated components exists a few lines before this part, this is after component creation.

The issue only is with template creation, which *is* unique per-integration for most integrations, due to differing dependencies.